plusminus/plusminus.py

```diff
@@ -1084,15 +1136,17 @@ def cvt_to_function_call(tokens):
 
         value_assignment_statement = (
             pp.delimitedList(lvalue)("lhs")
-            + pp.oneOf("<- =")
+            + pp.oneOf("<- = ←")
```

Comment:
I'd forgotten that I added this `<-` operator. I'm liking it less now that we are making the base set of operators. I might remove before 1.0. Leaving this in for now.